src/lib/moneyHelper.ts

```diff
@@ -0,0 +1,13 @@
+interface MoneyField {
+  amount: number
+  currencyCode: String
+}
+
+export const moneyFieldToUnit = (moneyField: MoneyField) => {
+  switch (moneyField.currencyCode) {
+    case "USD":
+      return moneyField.amount * 100
+    default:
+      return moneyField.amount
```

Comment:
This should imo throw an error, the developer should realise ASAP that the thing they’re trying to do is invalid.

Reply:
👍 whats the best way to through error?

Reply:
In this situation you should just throw one, there’s no need to handle this gracefully. `throw new Error(“...”)`